bin/bank/pycbc_brute_bank

```diff
@@ -46,7 +46,9 @@ parser.add_argument('--minimal-match', default=0.97, type=float)
 parser.add_argument('--buffer-length', default=2, type=float,
     help='size of waveform buffer in seconds')
 parser.add_argument('--cut-wavelength', type=bool, 
-                    help="Option to cut the wavelength")
+                    help="When enabled, the program will adjust the wavelength to fit the specified length in max-signal-length ")
```

Comment:
We could probably combine these two options, so if max-signal-length isn't given, it doesn't impose any cut.